src/mainwindow.cpp

```diff
@@ -320,7 +321,15 @@ void MainWindow::downloadModel(Model model) {
     ui_->cancelDownloadButton->setEnabled(true);
     showDownloadPane(true);
 
-    QNetworkReply *reply = network_.downloadFile(model.url, QCryptographicHash::Sha256, model.checksum);
+    // Pass on meta info about the model so we remember once we've downloaded it
+    ModelMeta meta;
+    meta.modelUrl = model.url;
+    meta.repositoryUrl = model.repositoryUrl;
+
+    qDebug() << "Downloading:";
```

Comment:
This should probably be removed, we don't want it:
`shortName: "en-bg-tiny" modelName: "English-Bulgarian tiny" url: "https://data.statmt.org/bergamot/models/bgen/enbg.student.tiny11.tar.gz" path: "" src: "English" trg: "Bulgarian" type: "tiny" localversion -1 localAPI -1 remoteversion: 1 remoteAPI 1
`